--- ./fast5-0.6.5/include/fast5/hdf5_tools.hpp.orig	2024-02-14 15:53:38.760440702 -0500
+++ ./fast5-0.6.5/include/fast5/hdf5_tools.hpp	2024-02-14 15:53:55.907734579 -0500
@@ -8,6 +8,7 @@
 #ifndef __HDF5_TOOLS_HPP
 #define __HDF5_TOOLS_HPP
 
+#include <array>
 #include <cassert>
 #include <cstring>
 #include <exception>
@@ -2106,7 +2107,11 @@
             detail::Util::wrap(H5Oopen, _file_id, loc_full_name.c_str(), H5P_DEFAULT),
             detail::Util::wrapped_closer(H5Oclose));
         H5O_info_t info;
+#if H5_VERSION_GE(1, 12, 0)
+        detail::Util::wrap(H5Oget_info, id_holder.id, &info, H5O_INFO_NUM_ATTRS);
+#else
         detail::Util::wrap(H5Oget_info, id_holder.id, &info);
+#endif
         // num_attrs in info.num_attrs
         for (unsigned i = 0; i < (unsigned)info.num_attrs; ++i)
         {
@@ -2386,7 +2391,11 @@
                 detail::Util::wrapped_closer(H5Oclose));
             // check object is a group
             H5O_info_t o_info;
+#if H5_VERSION_GE(1, 12, 0)
+            detail::Util::wrap(H5Oget_info, o_id_holder.id, &o_info, H5O_INFO_BASIC);
+#else
             detail::Util::wrap(H5Oget_info, o_id_holder.id, &o_info);
+#endif
             if (o_info.type != H5O_TYPE_GROUP) return false;
         }
         return true;
@@ -2407,7 +2416,11 @@
             detail::Util::wrapped_closer(H5Oclose));
         // check object is a group
         H5O_info_t o_info;
+#if H5_VERSION_GE(1, 12, 0)
+        detail::Util::wrap(H5Oget_info, o_id_holder.id, &o_info, H5O_INFO_BASIC);
+#else
         detail::Util::wrap(H5Oget_info, o_id_holder.id, &o_info);
+#endif
         return o_info.type == type_id;
     } // check_object_type()
 }; // class File
