--- pavilion2-7217d1f577653677dcc9d1708d3ecce326fa18b9/lib/pavilion/schedulers/plugins/slurm.py	2023-11-15 13:50:24.000000000 -0500
+++ pavilion2-fixed/lib/pavilion/schedulers/plugins/slurm.py	2024-01-21 23:09:22.337344393 -0500
@@ -291,9 +291,10 @@
                           'COMPLETING',
                           'PLANNED',
                           'MAINTENANCE',
+                          'MIXED',
                           'IDLE',
                           'MAINT'],
-            'avail_states': ['IDLE', 'MAINT', 'MAINTENANCE', 'PLANNED'],
+            'avail_states': ['IDLE', 'MIXED', 'MAINT', 'MAINTENANCE', 'PLANNED'],
             'features': [],
             'reserved_states': ['RESERVED'],
             'sbatch_extra': [],
