--- dakota-6.18.0-public-src-cli/src/unit/test_tolerance_intervals.cpp.orig	2023-08-22 14:30:38.352013715 -0400
+++ dakota-6.18.0-public-src-cli/src/unit/test_tolerance_intervals.cpp	2023-08-22 13:40:20.627422637 -0400
@@ -517,8 +517,8 @@
   // ************************************************************************
   BOOST_CHECK( num_valid_samples == 0 );
   for (size_t k = 0; k < num_fns; ++k) {
-    BOOST_CHECK( isnan(computed_dstien_mus   [k]) );
-    BOOST_CHECK( isnan(computed_dstien_sigmas[k]) );
+    BOOST_CHECK( std::isnan(computed_dstien_mus   [k]) );
+    BOOST_CHECK( std::isnan(computed_dstien_sigmas[k]) );
   }
 }
 
@@ -587,7 +587,7 @@
   BOOST_CHECK( num_valid_samples == 1 );
   for (size_t k = 0; k < num_fns; ++k) {
     BOOST_CHECK( computed_dstien_mus[k] == expected_dstien_mus[k] );
-    BOOST_CHECK( isnan(computed_dstien_sigmas[k]) );
+    BOOST_CHECK( std::isnan(computed_dstien_sigmas[k]) );
   }
 }
 
